starcheck/src/lib/Ska/Parse_CM_File.pm

```diff
@@ -65,30 +65,20 @@ sub dither {
 ###############################################################
     my $dh_file = shift;      # Dither history file name
     my $bs_arr = shift;               # Backstop array reference
+    my $kadi_dither = shift;
```

Comment:
Add a comment about where `kadi` comes from and the data structure.  Also add a comment for the whole function about what it does.  This function is really much more than parsing CM files.